data/build.gradle.kts

```diff
@@ -203,10 +203,10 @@ publishing {
 }
 
     signing {
-        val signingKeyId = secureProperties.getProperty("signing.keyId")
-        val signingKey = secureProperties.getProperty("signing.key")
-        val signingPassword = secureProperties.getProperty("signing.password")
+        val signingKeyId = System.getenv("SIGNING_KEY_ID")
```

Comment:
so for data you keep the signing but not for core?

Reply:
fixed